toolkit/modules/power_ops_template/workflows/bid_process_triggers.WorkflowTrigger.yaml

```diff
@@ -0,0 +1,102 @@
+- externalId: trigger_bid_process:bid_configuration_day_ahead_water_value_no2_full
+  triggerRule:
+    triggerType: schedule
+    cronExpression: "{{wf_day_ahead_bid_process_cron}}"
+  input:
+    bigConfiguration:
+      externalId: "bid_configuration_day_ahead_water_value_no2_full"
+    metadata:
+      config: "bid_configuration_day_ahead_water_value_no2_full"
+      market: "{{wf_day_ahead_market}}"
```

Comment:
**comment**: What do we need this for? Front-end, I guess? Not really related to this PR, but I would think this is redundant information, unless this is what the workflow uses (finding all workflow executions with metadata.market="DAY_AHEAD") to determine that this is a workflow execution supposed to produce a day-ahead bid? The combination of method, price area and config also seems redundant, given that the config contains that information (though it may require another API call). Probably best to discuss this with @notrixbe

Reply:
Yea for the front-end not an ideal solution, but @notrixbe and I have aligned on it